HPXMLtoOpenStudio/resources/hotwater_appliances.rb

```diff
@@ -809,8 +809,6 @@ def self.add_other_equipment(model, obj_name, space, design_level_w, frac_sens,
   end
 
   def self.add_water_use_equipment(model, obj_name, peak_flow, schedule, water_use_connections, mw_temp_schedule = nil)
-    return if peak_flow == 0.0
```

Comment:
If I don't remove this line, then the following don't get written, leading to "There are 1 unused schedules in input.".

```
WaterUse:Equipment,
  dhw fixtures,                           !- Name
  dhw fixtures,                           !- End-Use Subcategory
  0,                                      !- Peak Flow Rate {m3/s}
  dhw fixtures,                           !- Flow Rate Fraction Schedule Name
  mixed water temperature schedule;       !- Target Temperature Schedule Name

WaterUse:Equipment,
  dhw distribution waste,                 !- Name
  dhw distribution waste,                 !- End-Use Subcategory
  0,                                      !- Peak Flow Rate {m3/s}
  dhw fixtures,                           !- Flow Rate Fraction Schedule Name
  mixed water temperature schedule;       !- Target Temperature Schedule Name
```